src/kirby/components/form-field/_form-field-inputs.shared.scss

```
@@ -10,7 +10,7 @@
   border-radius: size('s');
   box-shadow: get-elevation(2);
   box-sizing: border-box;
-  color: get-color('black');
+  color: get-text-color('black');
```

Consider moving up next to  ` background-color: get-color('white');` and sync:
```suggestion
  color: get-color('black-contrast');
```